components/elements/button.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <button
+    :class="{
+      primary,
+      secondary,
+      tertiary
+    }"
+  >
+    <slot />
+  </button>
+</template>
+
+<script>
+export default {
+  props: {
+    primary: {
+      type: Boolean,
+      default: true
+    },
+    secondary: {
+      type: Boolean,
+      default: false
+    },
+    tertiary: {
+      type: Boolean,
+      default: false
+    }
+  }
+}
+</script>
+
+<style lang="postcss" scoped>
+button {
+  @apply rounded flex items-center px-6 py-3 text-sm font-semibold text-mtl-black-500 uppercase;
+}
+
+.primary {
+  @apply bg-mtl-green-500 text-mtl-white;
+}
+.secondary {
```

Comment:
NP: missing new line, maybe consider adding https://stylelint.io/?

```suggestion

.secondary {
```

Reply:
we will do it in a new pr as it's more for linting that for the design system itself